java-surefire/src/main/java/org/sonar/plugins/surefire/api/SurefireUtils.java

```diff
@@ -28,14 +28,28 @@
 import javax.annotation.CheckForNull;
 
 import java.io.File;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
 
 /**
  * @since 2.4
  */
 public final class SurefireUtils {
 
   private static final Logger LOGGER = Loggers.get(SurefireUtils.class);
+  /**
+   * @deprecated
+   * @since 4.11
```

Comment:
I don't think this is correct usage of `@since` , you can say that it was deprecated since 4.11 as a free text , reference https://docs.oracle.com/javase/1.5.0/docs/guide/javadoc/deprecation/deprecation.html#javadoc_tag